pkg/kubelet/cm/device_plugin_handler.go

```diff
@@ -217,6 +206,58 @@ func (h *DevicePluginHandlerImpl) Allocate(pod *v1.Pod, container *v1.Container,
 	return ret, nil
 }
 
+func (h *DevicePluginHandlerImpl) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAdmitResult {
+	pod := attrs.Pod
+	for _, container := range attrs.Pod.Spec.Containers {
+		for k, v := range container.Resources.Limits {
+			resource := string(k)
+			needed := int(v.Value())
+			if !deviceplugin.IsDeviceName(k) || needed == 0 {
```

Comment:
Could you take a look at the current code. This part has changed that we no longer use IsDeviceName checking because we currently don't have any naming convention just for device plugin resources.